### PR TITLE
feat(config): document config help

### PR DESCRIPTION
**Pull request by kindermoumoute**

### CODE REVIEW

File: internal/namespaces/config/commands.go [hunk 1]
```diff
@@ -29,10 +31,50 @@ func GetCommands() *core.Commands {
 }
 
 func configRoot() *core.Command {
+	configPath := scw.GetConfigPath()
+
 	return &core.Command{
-		Short:     `Config file management`,
-		Long:      `Manage your Scaleway CLI config file.`,
+		Short: `Config file management`,
+		Long: `Config management engine is common across all Scaleway developer tools (CLI, terraform, SDK,... ). It allows to handle your Scaleway config through two ways: environment variables or/and config file.
```
Comment (QuentinBrosse) [inline internal/namespaces/config/commands.go]: `SDK,... ` => `SDK, ...`

File: internal/namespaces/config/commands.go [hunk 2]
```diff
@@ -29,10 +31,50 @@ func GetCommands() *core.Commands {
...
+
+Scaleway config file is self-documented, we recommend you to have a look at it at least once before using Scaleway developer tools: ` + terminal.Style(configPath, color.Bold, color.FgBlue) + `
+
+In this CLI, ` + terminal.Style(`config file attributes are applied prior to the environment variables`, color.Bold) + `: environment variable > config attribute.
```
Comment (QuentinBrosse) [inline internal/namespaces/config/commands.go]: What do you think about `In this CLI, environment variables have priority over the configuration file.`?

File: internal/namespaces/config/commands.go [hunk 3]
```diff
@@ -29,10 +31,50 @@ func GetCommands() *core.Commands {
...
+
+The following environment variables are supported:
+- ` + terminal.Style("SCW_ACCESS_KEY", color.Bold, color.FgBlue) + ` the access key of a token
```
Comment (QuentinBrosse) [inline internal/namespaces/config/commands.go]: You use "your" bellow

File: internal/namespaces/config/commands.go [hunk 4]
```diff
@@ -29,10 +31,50 @@ func GetCommands() *core.Commands {
...
+- ` + terminal.Style("SCW_SECRET_KEY", color.Bold, color.FgBlue) + ` the secret key of a token
```
Comment (QuentinBrosse) [inline internal/namespaces/config/commands.go]: You use "your" bellow

File: internal/namespaces/config/commands.go [hunk 5]
```diff
@@ -29,10 +31,50 @@ func GetCommands() *core.Commands {
...
-		Short:     `Config file management`,
-		Long:      `Manage your Scaleway CLI config file.`,
+		Short: `Config file management`,
+		Long: `Config management engine is common across all Scaleway developer tools (CLI, terraform, SDK,... ). It allows to handle your Scaleway config through two ways: environment variables or/and config file.
+
+Scaleway config file is self-documented, we recommend you to have a look at it at least once before using Scaleway developer tools: ` + terminal.Style(configPath, color.Bold, color.FgBlue) + `
+
+In this CLI, ` + terminal.Style(`config file attributes are applied prior to the environment variables`, color.Bold) + `: environment variable > config attribute.
+
+The following environment variables are supported:
```
Comment (QuentinBrosse) [inline internal/namespaces/config/commands.go]: We should use a padding (cf. `tabwriter`) and capitalization:
```
The following environment variables are supported:
- SCW_ACCESS_KEY                Your access key of a token
- SCW_SECRET_KEY                Your secret key of a token
- SCW_DEFAULT_ORGANIZATION_ID   Your default organization ID
```

File: internal/namespaces/config/commands.go [hunk 6]
```diff
@@ -29,10 +31,50 @@ func GetCommands() *core.Commands {
...
+- ` + terminal.Style("SCW_ACCESS_KEY", color.Bold, color.FgBlue) + ` the access key of a token
+- ` + terminal.Style("SCW_SECRET_KEY", color.Bold, color.FgBlue) + ` the secret key of a token
+- ` + terminal.Style("SCW_DEFAULT_ORGANIZATION_ID", color.Bold, color.FgBlue) + ` your default organization ID
+- ` + terminal.Style("SCW_DEFAULT_REGION", color.Bold, color.FgBlue) + ` your default region
+- ` + terminal.Style("SCW_DEFAULT_ZONE", color.Bold, color.FgBlue) + ` your default availability zone
+- ` + terminal.Style("SCW_API_URL", color.Bold, color.FgBlue) + ` URL of the API
+- ` + terminal.Style("SCW_INSECURE", color.Bold, color.FgBlue) + ` set this to true to enable the insecure mode
+- ` + terminal.Style("SCW_PROFILE", color.Bold, color.FgBlue) + ` set the config profile to use
+
```
Comment (QuentinBrosse) [inline internal/namespaces/config/commands.go]: Maybe we should specify that the access/secret keys and the organization id can be found in the [Scaleway console](https://console.scaleway.com/account/credentials)?

File: internal/namespaces/config/commands.go [hunk 7]
```diff
@@ -29,10 +33,57 @@ func GetCommands() *core.Commands {
...
+		{"SCW_PROFILE", "Set the config profile to use"},
+	} {
+		fmt.Fprintf(w, "  %s\t%s\n", terminal.Style(envVar[0], color.Bold, color.FgBlue), envVar[1])
+	}
+	w.Flush()
 	return &core.Command{
-		Short:     `Config file management`,
-		Long:      `Manage your Scaleway CLI config file.`,
+		Short: `Config file management`,
+		Long: `Config management engine is common across all Scaleway developer tools (CLI, terraform, SDK, ... ). It allows to handle Scaleway config through two ways: environment variables or/and config file.
```
Comment (loicbourgois) [inline internal/namespaces/config/commands.go]: `or/and` -> `and/or`

File: internal/namespaces/config/commands.go [hunk 8]
```diff
@@ -29,10 +33,57 @@ func GetCommands() *core.Commands {
...
+
+Scaleway config file is self-documented, we recommend you to have a look at it at least once before using Scaleway developer tools: ` + terminal.Style(configPath, color.Bold, color.FgBlue) + `
```
Comment (loicbourgois) [inline internal/namespaces/config/commands.go]: `self-documented, we` -> `... is self-documented. We ...`

File: internal/namespaces/config/commands.go [hunk 9]
```diff
@@ -29,10 +33,57 @@ func GetCommands() *core.Commands {
 }
 
 func configRoot() *core.Command {
+	configPath := scw.GetConfigPath()
+	envVarTable := bytes.Buffer{}
+	w := tabwriter.NewWriter(&envVarTable, 5, 1, 2, ' ', tabwriter.ANSIGraphicsRendition)
+	for _, envVar := range [][2]string{
+		{"SCW_ACCESS_KEY", "The access key of a token (create a token on https://console.scaleway.com/account/credentials)"},
```
Comment (loicbourgois) [inline internal/namespaces/config/commands.go]: create a token `on` -> `at`

File: internal/namespaces/config/commands.go [hunk 10]
```diff
@@ -29,10 +33,57 @@ func GetCommands() *core.Commands {
...
+		{"SCW_SECRET_KEY", "The secret key of a token (create a token on https://console.scaleway.com/account/credentials)"},
```
Comment (loicbourgois) [inline internal/namespaces/config/commands.go]: create a token `on` -> `at`

File: internal/namespaces/config/commands.go [hunk 11]
```diff
@@ -29,10 +33,57 @@ func GetCommands() *core.Commands {
...
+		Long: `Config management engine is common across all Scaleway developer tools (CLI, terraform, SDK, ... ). It allows to handle Scaleway config through two ways: environment variables or/and config file.
+
+Scaleway config file is self-documented, we recommend you to have a look at it at least once before using Scaleway developer tools: ` + terminal.Style(configPath, color.Bold, color.FgBlue) + `
+
+In this CLI, ` + terminal.Style(`environment variables have priority over the configuration file`, color.Bold) + `.
+
+The following environment variables are supported:
+` + envVarTable.String() + `
+
+Read more about the config management engine on https://github.com/scaleway/scaleway-sdk-go/tree/master/scw#scaleway-config`,
```
Comment (loicbourgois) [inline internal/namespaces/config/commands.go]: engine `on` -> `at`

Comment (loicbourgois) [inline internal/namespaces/config/commands.go]: Should end sentence with a dot `.`

Comment (kindermoumoute) [reply internal/namespaces/config/commands.go]: let's avoid bad copy/pasta by ignoring this nit grammar

File: internal/namespaces/config/commands.go [hunk 12]
```diff
@@ -29,10 +33,57 @@ func GetCommands() *core.Commands {
...
+		{"SCW_PROFILE", "Set the config profile to use"},
+	} {
+		fmt.Fprintf(w, "  %s\t%s\n", terminal.Style(envVar[0], color.Bold, color.FgBlue), envVar[1])
+	}
+	w.Flush()
 	return &core.Command{
-		Short:     `Config file management`,
-		Long:      `Manage your Scaleway CLI config file.`,
+		Short: `Config file management`,
+		Long: `Config management engine is common across all Scaleway developer tools (CLI, terraform, SDK, ... ). It allows to handle Scaleway config through two ways: environment variables and/or config file.
```
Comment (QuentinBrosse) [inline internal/namespaces/config/commands.go]: Can you try with [RemoveIndent](https://github.com/scaleway/scaleway-cli/blob/v2/internal/interactive/print.go#L66)?

**Pull request merged**